tools/src/test/resources/1.5/valid-compositions-1.5.xml

```diff
@@ -41,7 +41,7 @@
                 <dependency ref="acme-application-1.0"/>
             </dependencies>
         </composition>
-        <composition>
+        <composition bom-ref="composition-2">
```

Comment:
please revert this line.
we need a showcase where no (optional) `bom-ref` attribute is set

Reply:
Good catch. Updated test cases.